docs/raspberry-pi_install.md

```diff
@@ -90,10 +90,10 @@ $ python3 config_generator.py
 ```
 - Once configured, you may run SUSI by executing the following command
 ```
-$ python3 -m main
+$ python3 app.py
```

Comment:
Write both commands for running with or without UI.

Reply:
ok, i will correct that